.gitignore

```diff
@@ -93,6 +93,10 @@ doc/doxygen/latex/*
 doc/doxygen/xml/*
 doc/doxygen/html/*
 
+# Cmake artifacts
+
+/CMakeFiles/
```

Comment:
This is not necessary. There is a bug that will be addressed by a separate PR and if you try to do an in-tree build you will have more than just these files polluting the source tree.

Reply:
Gotcha, reverted.